catalog/spatial/kml/spatial-kml-networklinkendpoint/pom.xml

```diff
@@ -82,7 +82,7 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>JavaAPIforKml, catalog-core-api-impl, handlebars,
-                            antlr4-runtime, commons-lang3, common-system, platform-util
+                            antlr4-runtime, commons-lang3, platform-util
```

Comment:
nit : since we are touching this, might be best to make the dependencies one per line

Reply:
Reformatted